osc.js

```diff
@@ -58,40 +184,53 @@ class OSCInstance extends InstanceBase {
 				],
 				default: 'udp',
 				width: 4
+			},
+			{
+				type: 'checkbox',
+				id: 'listen',
+				label: 'Listen for Feedback',
+				width: 4,
+				default: false,
+			},
+			{
+				type: 'textinput',
+				id: 'feedbackPort',
+				label: 'Feedback Port',
+				width: 4,
+				regex: Regex.PORT,
+				isVisible: (options, data) => (options.listen && options.protocol === 'udp'),
 			}
-
 		]
 	}
 
 	updateActions() {
 		const sendOscMessage = async (path, args) => {
-			this.log('debug', `Sending OSC [${this.config.protocol}] ${this.config.host}:${this.config.port} ${path}`)
+			this.log('debug', `Sending OSC [${this.config.protocol}] ${this.targetHost}:${this.config.targetPort} ${path}`)
 			this.log('debug', `Sending Args ${JSON.stringify(args)}`)
 
 			if (this.config.protocol === 'udp') {
-				this.oscSend(this.config.host, this.config.port, path, args);
+				this.oscSend(this.targetHost, this.config.targetPort, path, args);
 
 			} else if (this.config.protocol === 'tcp') {
 				
-				await this.tcpClient.sendCommand(path, args)
+				this.tcpClient.sendCommand(path, args)
 				.then(() => {
 					this.log('info', `TCP Command sent successfully. Path: ${path}, Args: ${JSON.stringify(args)}`);
 				})
 				.catch(err => {
-					this.log('error', 'Failed to send TCP command:', err);
+					this.log('error', 'Failed to send TCP command:', err.message);
 				});
 
 			} else if (this.config.protocol === 'tcp-raw') {
 
-				await this.rawClient.sendCommand(path, args)
+				this.rawClient.sendCommand(path, args)
```

Comment:
any reason you removed the `await` on these?
I think it is a good idea to have, so that companion can know when the send completed

Reply:
This is resolved in latest build